app/assets/javascripts/discourse/app/components/directory-table.js

```diff
@@ -2,16 +2,88 @@ import Component from "@ember/component";
 import { action } from "@ember/object";
 
 export default Component.extend({
-  classNames: ["directory-table-container"],
+  lastScrollPosition: 0,
+  ticking: false,
+  topHorizontalScrollBar: null,
+  tableContainer: null,
+
+  didInsertElement() {
+    this._super(...arguments);
+    this.set(
```

Comment:
Since `tableContainer` cannot be observed or part of a computed property, you might as well use `this._tableContainer = `